### ISSUE TITLE
Rework docs to use polyglot notebooks

### ISSUE THREAD

**Issue by kMutagene** [plotly/Plotly.NET#442]
Since fsdocs now supports generation of documentation with notebooks as input, a rework of the docs that also offers side-by-side examples for F# and C# is overdue.

I think this should be done gradually with a mechanism that allows publishing the new and old docs together as long as not all pages are ported.

This also would be a huge step forward for #376 , at least in a way that we could provide the docs in a format that is needed for the official pipeline.

**Comment by EverybodyKurts** [comment]
Would this be a good first pull request? I think this could be fun :)

**Comment by kMutagene** [comment]
@EverybodyKurts Absolutely, i think it would serve as a great first PR as you are learning using the library along the way of porting those samples. Note this includes updating to [fsdocs v20](https://www.nuget.org/packages/fsdocs-tool/) as the current version used does not support notebook input. This will not play nicely with current docs in my experience, so i think you could reduce the complexity of the task by 'just' porting some samples initially without the overhead of getting docs to render. If you know your way around fsdocs though, go ahead! thanks for contributing ;)

**Comment by Eddie-Hartman** [comment]
As a passerby, I just wanted to add that I'm definitely passing on this library due to a lack of C# documentation or examples. Just compare the two: https://plotly.com/csharp/ and https://plotly.com/fsharp/

I'm looking for a straightforward library to use in C# interactive notebooks and this looked promising, but the lack of docs and my inability to translate F# to C# is making me stay away for now.

**Comment by kMutagene** [comment]
@Eddie-Hartman fair points that are being worked on. It also does not help that the docs you linked are severely outated. This project is documented at https://plotly.net and although i tried contact them multiple times, there seems to be no interest to incorporate that into the existing stuff you linked at this time.

**Comment by Eddie-Hartman** [comment]
Yeah if plotly.net included docs with examples of how to do things in C# I'd certainly be happy enough to try it.

For example: https://plotly.net/distribution-charts/histograms.html

Thanks for the response and good luck!
